setup.py

```diff
@@ -48,7 +48,7 @@ def _dbt_synapse_version():
     packages=find_namespace_packages(include=['dbt', 'dbt.*']),
     include_package_data=True,
     install_requires=[
-        "dbt-core~={}".format(dbt_version),
-        "jaydebeapi",
+        f"dbt-core~={dbt_version}",
+        "firebolt-sdk>=0.1.0"
```

Comment:
we'll eventually bump this to the newer version right?

Reply:
Yeah, we will, but the SDK is moving fast, and I didn't know what version it would be when I didn't change it.

Reply:
```suggestion
        "firebolt-sdk>=0.2.0"
```